deploycala.py

```diff
@@ -205,6 +205,8 @@ def main():
     parser.add_argument("-i", "--incremental", action="store_true", dest="incremental",
                         help="do incremental builds, i.e. don't clear the build directory before building, if found")
     parser.add_argument("--noupdate", action="store_true", dest="noupdate", help=argparse.SUPPRESS)
+    parser.add_argument("-u", "--url", nargs=1, default=None, dest="url",
```

Comment:
maybe is better to set the default value to "https://github.com/calamares/calamares.git" and skip the 'if args.url: ' code below. (must be verified if the default value is set correctly by the parser if not specified by the user the argument "--url")